RR.App.Tests/Controllers/HRIS/ClientProjectsControllerUnitTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿using HRIS.Models;
+using HRIS.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using RR.App.Controllers.HRIS;
+using Xunit;
+
+namespace HRIS.Tests.Controllers;
+    public class ClientProjectsControllerUnitTests
+    {
+        private Mock<IClientProjectService> _mockClientProjectService;
+        private ClientProjectsController _controller;
+
+        public ClientProjectsControllerUnitTests()
+        {
+            _mockClientProjectService = new Mock<IClientProjectService>();
+            _controller = new ClientProjectsController(_mockClientProjectService.Object);
+        }
+
+        [Fact]
+        public async Task GetAllClientProjects_ReturnsOkResult()
+        {
+           var expectedClientProjects = new List<ClientProjectsDto>
+           {
+                new ClientProjectsDto
+                {
+                  Id = 1,
+                  EmployeeId = 1,
+                  ClientName = "string",
+                  ProjectName = "string",
+                  EndDate = DateTime.Now,
+                  StartDate = DateTime.Now,
+                  ProjectURL = "string",
+                }
+           };
```

Comment:
Okay so we have test data files now When you have more than one of the same type of object we use test data files